src/main/java/org/broad/igv/ui/action/LoadFromURLMenuAction.java

```diff
@@ -96,6 +98,20 @@ public void actionPerformed(ActionEvent e) {
                             MessageUtils.showMessage("Error loading url: " + url + " (" + ex.toString() + ")");
                         }
                     } else {
+                        try {
+                            // If AWS support is active, check if objects are in accessible tiers via Load URL menu...
+                            if (AmazonUtils.isAwsS3Path(url)) {
+                                String bucket = AmazonUtils.getBucketFromS3URL(url);
+                                String key = AmazonUtils.getKeyFromS3URL(url);
+
+                                AmazonUtils.s3ObjectAccessResult res = isObjectAccessible(bucket, key);
+                                if (!res.getObjAvailable()) { MessageUtils.showErrorMessage(res.getErrorReason(), null); return; }
+                            }
+                        } catch (NullPointerException npe) {
```

Comment:
Just a comment, do we know that the user hasn't logged in solely from the npe?  Could there be other reasons for an npe?